theme/js/script.js

```diff
@@ -44,46 +44,53 @@ $(document).ready(function() {
 
     countCharacter();
 
+    openDonationContent();
```

Comment:
You don’t need this line, you can remove it.

Reply:
done